.github/workflows/build.yml

```diff
@@ -7,10 +7,10 @@ on: [push, pull_request]
 
 jobs:
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9]
```

Comment:
no python 3.9?

Reply:
tried with 3.9 but the pysam version isn't compatible with it